Kiss.Bff.Test/PostKlantContactenCustomProxyTests.cs

```diff
@@ -0,0 +1,445 @@
+﻿using Moq;
+using Microsoft.Extensions.Configuration;
+using System.Text.Json.Nodes;
+using Kiss.Bff.ZaakGerichtWerken.Contactmomenten;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+using System.Net;
+using System.Text.Json;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+
+namespace Kiss.Bff.Test
+{
+    [TestClass]
+    public class PostKlantContactenCustomProxyTests
+    {
+        private Mock<IConfiguration> _configurationMock;
+        private Mock<GetMedewerkerIdentificatie> _getMedewerkerIdentificatieMock;
+        private Mock<IAuthenticationHeaderProvider> _authProviderMock;
+        private PostKlantContactenCustomProxy _controller;
+        private DefaultHttpContext _httpContext;
+
+        [TestInitialize]
+        public void Setup()
+        {
```

Comment:
Dit ziet er best ingewikkeld uit. Heb je de library overwogen die we elders ook gebruiken?
https://github.com/richardszalay/mockhttp